moskito-webui/src/main/java/net/anotheria/moskito/webui/auth/actions/LoginAction.java

```diff
@@ -0,0 +1,33 @@
+package net.anotheria.moskito.webui.auth.actions;
+
+import net.anotheria.maf.action.Action;
+import net.anotheria.maf.action.ActionCommand;
+import net.anotheria.maf.action.ActionMapping;
+import net.anotheria.maf.bean.FormBean;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class LoginAction implements Action{
+
+    @Override
+    public void preProcess(ActionMapping mapping, HttpServletRequest req, HttpServletResponse res) throws Exception {
+
+    }
+
+    @Override
+    public ActionCommand execute(ActionMapping mapping, FormBean formBean, HttpServletRequest req, HttpServletResponse res) throws Exception {
+        req.setAttribute("title", "Log in Moskito-Inspect");
+
+        if("1".equals(req.getParameter("error")))
```

Comment:
thats ok, why would you put a "1" in the request to indicate an error? It would be much easier to use Boolean.TRUE. Also you don't need to check the value, the presence is sufficient.